returnn/datasets/generating.py

```diff
@@ -13,6 +14,8 @@
 import sys
 import typing
 
+from returnn.datasets.util.vocabulary import *
```

Comment:
This import code is inconsistent.
Why is there an empty line before?
And why is it not next to the other dataset related imports?
Also, again I would make it a relative import.